website/content/api-docs/system/plugins-runtimes-catalog.mdx

```diff
@@ -0,0 +1,177 @@
+---
+layout: api
+page_title: /sys/plugins/runtimes/catalog - HTTP API
+description: The `/sys/plugins/runtimes/catalog` endpoint is used to manage plugin runtimes.
+---
+
+# `/sys/plugins/runtimes/catalog`
+
+The `/sys/plugins/runtimes/catalog` endpoint is used to read, register, update, and
+remove plugin runtimes in Vault's catalog. Plugin runtimes must be registered before use, and
+once registered, backends can use the plugin runtime by querying the catalog.
+
+## LIST plugin runtimes
+
+This endpoint lists the plugin runtimes in the catalog.
+
+- **`sudo` required** – This endpoint requires `sudo` capability in addition to
+  any path-specific capabilities.
+
+| Method | Path                   |
+| :----- | :--------------------- |
+| `LIST`  | `/sys/plugins/runtimes/catalog` |
```

Comment:
It looks like either `LIST` or `GET` works for this endpoint, though since the plugins/catalog endpoint doc uses `GET`, perhaps we should use `GET` here as well? 

```suggestion
| `GET`  | `/sys/plugins/runtimes/catalog` |
```

Reply:
The plugins/catalog endpoint has its own additional baggage, I would just list two lines here, one for LIST and one for GET.